Homework4/Homework4.cs

```diff
@@ -0,0 +1,206 @@
+            {
+                var multiplicationResult = number3 * numberForMultiplication;
+                Console.WriteLine($"{number3} * {numberForMultiplication} = {multiplicationResult}");
+                numberForMultiplication++;
+            }
+        }
+
+        /* Task 3
+        Создать массив чисел [3, 5, 9, 8, 15]. Найдите произведение этих чисел и вывести в консоль результат.
+        */
+
+        private static void Task3()
+        {
+            Console.WriteLine("Task 3");
+            int[] ints = new int[] { 3, 5, 9, 8, 15 };
+            var multiplicationResult = 1;
+            foreach (int i in ints)
+            {
+                multiplicationResult *= i;
+            }
+            Console.WriteLine($"Multiplication result for numbers in [3, 5, 9, 8, 15] array: {multiplicationResult}");
+        }
+
+        /* Task 4
+        Написать программу, которая подсчитывает, сколько раз необходимо поделить число 2048 на 2, чтобы оно стало меньше 10.
+        */
+
+        private static void Task4()
+        {
+            Console.WriteLine("Task 4");
+            var num = 2048;
+            var numberOfDivisions = 0;
+            while (num > 10)
+            {
+                num /= 2;
+                numberOfDivisions++;
+            }
+            Console.WriteLine($"The number of divisions for 2048 till it becomes less than 10 is: {numberOfDivisions}");
+        }
+
+        /* Task 5
+        Создать массив строк. Написать программу, которая анализирует созданный массив и в случае наличие строки со значением “Hello”,
+        выводит в консоль слово “Labas!” и выходит из цикла.
+        */
+
+        private static void Task5()
+        {
+            Console.WriteLine("Task 5");
+            string[] namesOfHogwartsStudents = new string[] { "Harry", "Ron", "Neville", "Hello", "Hermione", "Pansy" };
+            foreach (var item in namesOfHogwartsStudents)
+            {
+                if (item == "Hello")
+                {
+                    Console.WriteLine("Labas!");
+                    break;
+                }
+            }
+        }
+
+        /* Task 6
+         Создайте массив чисел. Напишите программу, которая подсчитывает сумму первого
+        и последнего элемента массива и выводит ее в консоль.
+         */
+
+        private static void Task6()
+        {
+            Console.WriteLine("Task 6");
+            int[] ints = new int[] { 10, 3, 5, 9, 8, 15, 45 };
+            var summ = ints[0] + ints[ints.Length - 1];
+            Console.WriteLine($"{ints.First()} + {ints.Last()} = {summ}");
+        }
+
+        /* Task 7
+         Найдите сумму индексов минимального и максимального элементов массива.
+         */
+
+        private static void Task7()
+        {
+            Console.WriteLine("Task 7");
+            int[] ints = new int[] { 1, 0, 5, 9, 8, 99, 15, 7 };
+            var numOfMax = 0;
+            var numOfMin = 0;
+
+            foreach (var item in ints)
```

Comment:
Yep, it works but a bit complicated. You can create initialize numOfMin and numOfMax with the first element of the array and use 1 for starting with the second element. I;m approving as it is now